_data/processes.yml

```diff
@@ -14,6 +14,7 @@
 "globalShortcut": "Main Process"
 "ipcMain": "Main Process"
 "ipcRenderer": "Renderer Process"
+"Locales": "Main Process"
```

Comment:
I think the ` "Possible Values of app.getLocale()": "Main Process"` line can now be removed from this file.